src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml

```diff
@@ -143,3 +143,11 @@ ra_location:
     requirements:
         raLocationId: ".+"
     condition: "request.headers.get('Accept') matches '/^application\\\\/json($|[;,])/'"
+
+institution_configuration_options_for_institution:
+    path:    /institution-configuration-options/{institution}
```

Comment:
no sure how I feel about this URL... not sure that I believe that a resource should exist for an institution configuration options... especially with keeping ra-locations in mind. Dunno, it just feels off... can't exactly think of something obviously better, but yeah...